docs/configs/pdf/index.rst

```diff
@@ -11,7 +11,7 @@ Getting started
    :titlesonly:
    :maxdepth: 2
 
-   Installing the SDK <getting-started/installation>
+   Installing DAML Connect <getting-started/installation>
```

Comment:
Given that we renamed the thing installed by the assistant to "DAML SDK", I think this should also be DAML SDK.